src/core/public/i18n/i18n_eui_mapping.tsx

```diff
@@ -9,14 +9,13 @@
 import React from 'react';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
-import { EuiTokensObject } from '@elastic/eui';
 
 interface EuiValues {
   [key: string]: any;
 }
 
 export const getEuiContextMapping = () => {
-  const euiContextMapping: EuiTokensObject = {
+  return {
```

Comment:
Should we add `EuiTokensObject` to the return type for this function to ensure it matches?

```diff
-export const getEuiContextMapping = () => {
+export const getEuiContextMapping = (): EuiTokensObject => {
```